jobsdb/setup.go

```diff
@@ -37,13 +37,12 @@ func (jd *HandleT) setupDatabaseTables(l lock.DSListLockToken, clearAll bool) {
 		"Prefix":   jd.tablePrefix,
 		"Datasets": datasetIndices,
 	}
-
 	psqlInfo := GetConnectionString()
+	var err error
```

Comment:
minor: Need this?

Reply:
nope, leftover since the time I was fighting with parallel tests, thanks for spotting!